<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="com.demo.web.rest.errors.ExceptionTranslatorIntTest" tests="11" skipped="0" failures="0" errors="0" timestamp="2018-12-22T09:24:04" hostname="LAPTOP-ANBQLHRS" time="1.193">
  <properties/>
  <testcase name="testMissingServletRequestPartException" classname="com.demo.web.rest.errors.ExceptionTranslatorIntTest" time="0.119"/>
  <testcase name="testMissingServletRequestParameterException" classname="com.demo.web.rest.errors.ExceptionTranslatorIntTest" time="0.109"/>
  <testcase name="testAccessDenied" classname="com.demo.web.rest.errors.ExceptionTranslatorIntTest" time="0.115"/>
  <testcase name="testParameterizedError" classname="com.demo.web.rest.errors.ExceptionTranslatorIntTest" time="0.122"/>
  <testcase name="testInternalServerError" classname="com.demo.web.rest.errors.ExceptionTranslatorIntTest" time="0.114"/>
  <testcase name="testMethodNotSupported" classname="com.demo.web.rest.errors.ExceptionTranslatorIntTest" time="0.115"/>
  <testcase name="testExceptionWithResponseStatus" classname="com.demo.web.rest.errors.ExceptionTranslatorIntTest" time="0.098"/>
  <testcase name="testMethodArgumentNotValid" classname="com.demo.web.rest.errors.ExceptionTranslatorIntTest" time="0.106"/>
  <testcase name="testParameterizedError2" classname="com.demo.web.rest.errors.ExceptionTranslatorIntTest" time="0.097"/>
  <testcase name="testUnauthorized" classname="com.demo.web.rest.errors.ExceptionTranslatorIntTest" time="0.097"/>
  <testcase name="testConcurrencyFailure" classname="com.demo.web.rest.errors.ExceptionTranslatorIntTest" time="0.096"/>
  <system-out><![CDATA[2018-12-22 17:24:04.605  WARN 16740 --- [    Test worker] o.z.problem.spring.common.AdviceTrait    : Bad Request: Required request part 'part' is not present
2018-12-22 17:24:04.716  WARN 16740 --- [    Test worker] o.z.problem.spring.common.AdviceTrait    : Bad Request: Required String parameter 'param' is not present
2018-12-22 17:24:04.830  WARN 16740 --- [    Test worker] o.z.problem.spring.common.AdviceTrait    : Forbidden: test access denied!
2018-12-22 17:24:04.949  WARN 16740 --- [    Test worker] o.z.problem.spring.common.AdviceTrait    : Bad Request: Parameterized Exception
2018-12-22 17:24:05.054 ERROR 16740 --- [    Test worker] o.z.problem.spring.common.AdviceTrait    : Internal Server Error

java.lang.RuntimeException: null
	at com.demo.web.rest.errors.ExceptionTranslatorTestController.internalServerError(ExceptionTranslatorTestController.java:64)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.springframework.web.method.support.InvocableHandlerMethod.doInvoke(InvocableHandlerMethod.java:209)
	at org.springframework.web.method.support.InvocableHandlerMethod.invokeForRequest(InvocableHandlerMethod.java:136)
	at org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:102)
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.invokeHandlerMethod(RequestMappingHandlerAdapter.java:891)
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.handleInternal(RequestMappingHandlerAdapter.java:797)
	at org.springframework.web.servlet.mvc.method.AbstractHandlerMethodAdapter.handle(AbstractHandlerMethodAdapter.java:87)
	at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:991)
	at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:925)
	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:974)
	at org.springframework.web.servlet.FrameworkServlet.doGet(FrameworkServlet.java:866)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:687)
	at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:851)
	at org.springframework.test.web.servlet.TestDispatcherServlet.service(TestDispatcherServlet.java:71)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
	at org.springframework.mock.web.MockFilterChain$ServletFilterProxy.doFilter(MockFilterChain.java:166)
	at org.springframework.mock.web.MockFilterChain.doFilter(MockFilterChain.java:133)
	at org.springframework.test.web.servlet.MockMvc.perform(MockMvc.java:165)
	at com.demo.web.rest.errors.ExceptionTranslatorIntTest.testInternalServerError(ExceptionTranslatorIntTest.java:143)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.springframework.test.context.junit4.statements.RunBeforeTestExecutionCallbacks.evaluate(RunBeforeTestExecutionCallbacks.java:73)
	at org.springframework.test.context.junit4.statements.RunAfterTestExecutionCallbacks.evaluate(RunAfterTestExecutionCallbacks.java:83)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.springframework.test.context.junit4.statements.RunBeforeTestMethodCallbacks.evaluate(RunBeforeTestMethodCallbacks.java:75)
	at org.springframework.test.context.junit4.statements.RunAfterTestMethodCallbacks.evaluate(RunAfterTestMethodCallbacks.java:86)
	at org.springframework.test.context.junit4.statements.SpringRepeat.evaluate(SpringRepeat.java:84)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:251)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:97)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:70)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:190)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.runTestClass(JUnitTestClassExecutor.java:106)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.execute(JUnitTestClassExecutor.java:58)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.execute(JUnitTestClassExecutor.java:38)
	at org.gradle.api.internal.tasks.testing.junit.AbstractJUnitTestClassProcessor.processTestClass(AbstractJUnitTestClassProcessor.java:66)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:32)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
	at com.sun.proxy.$Proxy2.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.processTestClass(TestWorker.java:117)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:155)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:137)
	at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:404)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
	at java.lang.Thread.run(Thread.java:745)

2018-12-22 17:24:05.183  WARN 16740 --- [    Test worker] o.z.problem.spring.common.AdviceTrait    : Method Not Allowed: Request method 'POST' not supported
2018-12-22 17:24:05.282  WARN 16740 --- [    Test worker] o.z.problem.spring.common.AdviceTrait    : Bad Request: null
2018-12-22 17:24:05.388  WARN 16740 --- [    Test worker] o.z.problem.spring.common.AdviceTrait    : Bad Request: Validation failed for argument at index 0 in method: public void com.demo.web.rest.errors.ExceptionTranslatorTestController.methodArgument(com.demo.web.rest.errors.ExceptionTranslatorTestController$TestDTO), with 1 error(s): [Field error in object 'testDTO' on field 'test': rejected value [null]; codes [NotNull.testDTO.test,NotNull.test,NotNull.java.lang.String,NotNull]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [testDTO.test,test]; arguments []; default message [test]]; default message [must not be null]] 
2018-12-22 17:24:05.487  WARN 16740 --- [    Test worker] o.z.problem.spring.common.AdviceTrait    : Bad Request: Parameterized Exception
2018-12-22 17:24:05.585  WARN 16740 --- [    Test worker] o.z.problem.spring.common.AdviceTrait    : Unauthorized: test authentication failed!
2018-12-22 17:24:05.681  WARN 16740 --- [    Test worker] o.z.problem.spring.common.AdviceTrait    : Conflict: test concurrency failure
]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
